### PR TITLE
More launch config

### PR DESCRIPTION
**Pull request by cavis**
Parts of #62.

- [x] Larger storage for prod (including setting dm.basesize)
- [x] Cloudwatch metrics for docker disk usage 
- [x] Reclaim (fstrim) docker disk space

### CODE REVIEW

File: stacks/ecs.yml [hunk 1]
```diff
@@ -283,12 +319,33 @@ Resources:
...
+            - |
+              Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+              MIME-Version: 1.0
+              --==BOUNDARY==
+              Content-Type: text/cloud-boothook; charset="us-ascii"
+              cloud-init-per once docker_options echo 'OPTIONS="${!OPTIONS} --storage-opt dm.basesize=${BaseSize}G"' >> /etc/sysconfig/docker
+              --==BOUNDARY==
+              Content-Type: text/x-shellscript; charset="us-ascii"
+              yum install -y aws-cfn-bootstrap
+              /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource ECSClusterASGLaunchConfiguration --region ${AWS::Region}
```
Comment (cavis) [inline stacks/ecs.yml]: I _assume_ these `${AWS::StackName}` variables are still interpolated, even though I didn't specify them in the var mapping below.

File: stacks/ecs.yml [hunk 2]
```diff
@@ -283,12 +319,33 @@ Resources:
...
-          /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource ECSClusterASGLaunchConfiguration --region ${AWS::Region}
-          /opt/aws/bin/cfn-signal -e $? --stack ${AWS::StackName} --resource ECSClusterASG --region ${AWS::Region}
+        Fn::Base64:
+          Fn::Sub:
+            - |
+              Content-Type: multipart/mixed; boundary="==BOUNDARY=="
+              MIME-Version: 1.0
+              --==BOUNDARY==
+              Content-Type: text/cloud-boothook; charset="us-ascii"
+              cloud-init-per once docker_options echo 'OPTIONS="${!OPTIONS} --storage-opt dm.basesize=${BaseSize}G"' >> /etc/sysconfig/docker
```
Comment (farski) [inline stacks/ecs.yml]: Where is `!OPTIONS` coming from?

Comment (cavis) [inline stacks/ecs.yml]: I wanted the literal `OPTIONS=${OPTIONS}` echo'd into that file.  And apparently (according to the docs) CFN will translate `${!Literal}` -> `${Literal}`.

Comment (farski) [inline stacks/ecs.yml]: Interesting.

File: stacks/ecs.yml [hunk 3]
```diff
@@ -247,6 +250,11 @@ Resources:
               command: !Sub |
                 #!/bin/bash
                 echo ECS_CLUSTER=${ECSCluster} >> /etc/ecs/ecs.config
+                echo ECS_ENGINE_TASK_CLEANUP_WAIT_DURATION=10m >> /etc/ecs/ecs.config
+            02_cloudwatch_disk_usage:
+              command: "(crontab -uroot -l 2> /dev/null; echo '* * * * * /root/cwdiskusage.sh') | crontab -u root -"
+            03_fstrim_reclaim_disk:
+              command: "(crontab -uroot -l 2> /dev/null; echo '*/5 * * * * docker ps -qa | xargs docker inspect --format="{{ .State.Pid }}" | xargs -IZ /sbin/fstrim /proc/Z/root/') | crontab -u root -"
```
Comment (cavis) [inline stacks/ecs.yml]: Added a link to the docs, where they talk about debugging "out of space" errors.

**Pull request merged**